src/main/java/esi/backend/service/RequestService.java

```diff
@@ -50,16 +52,4 @@ public void deleteRequest(UUID id) {
         requestRepository.deleteById(id);
     }
 
-    public ResponseEntity<List<Request>> getAllRequestsByCustomerId(UserDetails currentUser, long customerId) {
```

Comment:
Why delete this method?

Reply:
Why tho... if it returns a list of requests, not customers? Shouldn't the methods with the types of objects returned be in their respective service classes?